docs/products/grafana/howto/replace-expression-string.rst

```diff
@@ -0,0 +1,102 @@
+Replacing string in marshalled dashboards for Grafana®
```

Comment:
Can we improve this title?

Is `marshalled dashboards` a specific technical term?

Would it be sufficient to say `Replacing strings in metric expressions in Grafana® dashboards` ?

(and is `metric expressions` the right term?)

Reply:
resolved

Reply:
well it means that we just take the json payload of the grafana dashboards and do pretty dumb string replacement there.

Reply:
so it does not replace them only in metric expressions, if you replace "}" by "]" then it will break the json structure and fail even.

Reply:
i think ```Replacing string in marshalled dashboards for Grafana®``` or some reformulation of it is more precise.